readme.md

```diff
@@ -147,33 +146,51 @@ The options page includes [fregante/webext-options-sync][link-options-sync] that
 
 Refer to the Node modules documentation for more info on how it works.
 
-### Auto-publishing
+### Publishing
 
-The included [Travis file](.travis.yml) includes config to test the repo on merges to master and includes as deplpyment script to create and publish a new version of the extension only if the following conditions are met
-
-1. If the Travis build is triggered using the "Trigger build" button on the Travis website, this allows for [manual releases](#manual-releases).
-2. Or if the Travis build is part of a cron job and only there are any commits in the last 26 hours.
-
-To setup this auto-publishing, you have to export some API keys from Chrome Web Store (CWS) and Mozilla Extention Store (AMO), and set these as environment variables in your Travis settings.
+It's possible to publish to both the Chrome Web Store and Mozilla Addons at once by creating these ENV variables:
 
 1. `CLIENT_ID`, `CLIENT_SECRET`, and `REFRESH_TOKEN` from [Google APIs][link-cws-keys].
 1. `WEB_EXT_API_KEY`, and `WEB_EXT_API_SECRET` from [AMO][link-amo-keys].
 
-And don't forget to setup a cron job that runs daily on master, and include your CWS extension id in [.travis.yml](.travis.yml).
+And then running:
+
+```sh
+npm run release
+```
+
+This will:
+
+1. Build the extension
```

Comment:
Changes these list numbers be all `1`s instead of `1` `2` `3`, to be consistent.

Reply:
Wouldn't be better to use `1` `2` `3` at all places?

Reply:
It would be harder to reorder and add/delete new list items, as it requires us to change all most of them.